modules/ww-testdata/README.md

```diff
@@ -237,6 +253,43 @@ Creates a test GDC manifest file containing small open-access files for testing
 **Outputs**:
 - `manifest` (File): GDC manifest file containing 3 small open-access TCGA files (total ~210KB)
 
+### download_shapemapper_data
+
+Downloads the official ShapeMapper example data (TPP riboswitch) from the Weeks-UNC/shapemapper2 repository. This dataset contains RNA structure probing data suitable for testing ShapeMapper analysis workflows.
+
+**Use Case**: When testing RNA structure analysis workflows with ShapeMapper, you need appropriate test data with actual chemical probing experiments. This task downloads the canonical TPP riboswitch example data that includes both modified (chemically treated) and untreated control samples.
+
+**Inputs**:
+- `cpu_cores` (Int): CPU allocation (default: 1)
+- `memory_gb` (Int): Memory allocation (default: 4)
+
+**Outputs**:
+- `target_fa` (File): Target RNA FASTA file (TPP riboswitch sequence, ~200 nucleotides)
+- `modified_r1` (File): R1 FASTQ file from modified/treated sample (TPPplus, concatenated from split files)
+- `modified_r2` (File): R2 FASTQ file from modified/treated sample (TPPplus, concatenated from split files)
+- `untreated_r1` (File): R1 FASTQ file from untreated control sample (TPPminus, concatenated from split files)
+- `untreated_r2` (File): R2 FASTQ file from untreated control sample (TPPminus, concatenated from split files)
+
+**Data Source**: https://github.com/Weeks-UNC/shapemapper2/tree/master/example_data
+
+**Example Usage**:
+```wdl
+# For ShapeMapper RNA structure analysis
+call testdata.download_shapemapper_data { }
+
+call shapemapper.run_shapemapper {
+  input:
+    sample_name = "TPP_riboswitch",
+    target_fa = download_shapemapper_data.target_fa,
+    modified_r1 = download_shapemapper_data.modified_r1,
+    modified_r2 = download_shapemapper_data.modified_r2,
+    untreated_r1 = download_shapemapper_data.untreated_r1,
+    untreated_r2 = download_shapemapper_data.untreated_r2,
+    is_amplicon = true,
+    min_depth = 1000
+}
```

Comment:
Most tasks don't have this kind of example, I would consider leaving it out for consistency (unless we want to start including examples like this)